src/main/java/com/musicshare/controller/PlaylistController.java

```diff
@@ -20,13 +20,19 @@ public PlaylistController(PlaylistService service) {
         this.service = service;
     }
 
-    @PostMapping("/{playListName}")
-    public PlayListResponse createPlayList(@PathVariable String playListName) {
-
-        PlayListResponse response = PlayListResponse.builder()
-                .message("playlist created successfully.")
-                .data(service.createPlaylist(playListName))
-                .build();
+    @PostMapping
+    public PlayListResponse createPlayList(@RequestParam("playListName") String playListName) {
+        PlayListResponse response = null;
+        if (StringUtils.hasText(playListName)) {
```

Comment:
we can have two url paths defined here with one returning error message

Reply:
updated.